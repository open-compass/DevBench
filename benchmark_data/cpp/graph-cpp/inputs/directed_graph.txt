1
1 2 1
1 4 4
2 4 2
2 3 9
3 1 3
3 2 5
3 4 8
4 3 6