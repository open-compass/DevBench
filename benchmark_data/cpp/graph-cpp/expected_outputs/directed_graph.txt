Adjacency Matrix: 
0 0 0 0 0 
0 0 1 0 4 
0 0 0 9 2 
0 3 5 0 8 
0 0 0 6 0 
Adjacency Table: 
0	1	->	1	1	->	2	4	->	^	
1	2	->	2	2	->	3	9	->	^	
2	4	->	3	6	->	^	
3	3	->	0	3	->	1	5	->	2	8	->	^	
Depth-first search (no recursion) 
1 2 4 3 
BFS: 
1 2 4 3 
